etcdclient/api.go

```diff
@@ -13,8 +13,7 @@ import (
 )
 
 const (
-	certPath      = "/etc/ssl/etcd/ssl/"
-	trustedCAFile = certPath + "ca.pem"
+	trustedCAFile = "/ca.pem"
```

Comment:
I believe you should not start the filename with a slash... just prepend it (i.e. append trustedCAFile to the path)

Reply:
Normally, I would not want this.    

However, I was worried about someone coming along later and removing the trailing "/" from certPath.   This is defensive programming.

Reply:
@edmc-ss Do you understand why I did that?

Reply:
I'm just saying you have this backwards... :-)
How about this... rename the field "certPath" to "certDir"... so it is explicit it is a dir
Then, if the user specifies the trailing slash, it wouldn't matter if, instead, you computed your path by:
   certDir + "/" + trustedCAFile
Filenames don't have slashes...

Reply:
Okay - I changed all places from "certPath" to "certDir" so that it is clearer

Let me know what you think

Reply:
Ah... but you left the leading '/' on trustedCAFile's value :-)

It's just a pet peive of mine... I can live with it this way...